src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -120,16 +120,16 @@
         <button class="btn btn--red" on:click={showRemovePaymentCardDialog}>Remove</button>
       {/if}
 
-      <button class="btn-2 mrg-l mrg--l" on:click={() => showUpdatePaymentCardDialog()}>
+      <button class="btn-2 second-btn" on:click={() => showUpdatePaymentCardDialog()}>
```

Comment:
`second-btn` is too vague. Better to name `payment-btn` since it's only related to the payment card element